controllers/runtimecomponent_controller.go

```diff
@@ -234,6 +234,13 @@ func (r *RuntimeComponentReconciler) Reconcile(ctx context.Context, req ctrl.Req
 		reqLogger.V(1).Info(fmt.Sprintf("%s is not supported on the cluster", servingv1.SchemeGroupVersion.String()))
 	}
 
+	// Check if there is an existing Deployment, Statefulset or Knative service by this name
```

Comment:
To avoid processing the service account above, perform this check earlier (suggest [here](https://github.com/application-stacks/runtime-component-operator/blob/671d91fff45bc97996a66891c659882bd7ad58a0/controllers/runtimecomponent_controller.go#L137))